### PR TITLE
Notice of deprecation for uws, added alternative

### PR DESCRIPTION
**Pull request by DarkGhostHunter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -321,12 +319,15 @@ add a script tag to your html like so:
 
 _Note: When using the socket.io client library from your running server, remember to check that the `io` global variable is defined before subscribing to events._
 
-#### Better performance with [µWebSockets](https://github.com/uWebSockets/uWebSockets)
-For extra performance, you can use the faster `uws` engine instead of `ws`, by setting the `wsEngine` option for Socket.IO in `laravel-echo-server.json`:
+#### µWebSockets deprecation
+
+µWebSockets has been [officially deprecated](https://www.npmjs.com/package/uws). Currently there is no support for µWebSockets in Socket.IO, but it may have the new [ClusterWS](https://www.npmjs.com/package/@clusterws/cws) support incoming.
+
+Meanwhile, you can use the [`ws` engine](https://www.npmjs.com/package/ws), by setting the `wsEngine` option for Socket.IO in `laravel-echo-server.json`:
```
Comment (tlaverdure) [inline README.md]: This is the default engine used so no need to mention this.

File: README.md [hunk 2]
```diff
@@ -321,12 +319,15 @@ add a script tag to your html like so:
...
 
 ```js
```
Comment (tlaverdure) [inline README.md]: Let's remove this example for the time being.

**Comment by tlaverdure** [comment]
Thanks for the PR.

Closes https://github.com/tlaverdure/laravel-echo-server/issues/380

File: README.md [hunk 3]
```diff
@@ -321,14 +321,4 @@ _Note: When using the socket.io client library from your running server, remembe
...
-Meanwhile, you can use the [`ws` engine](https://www.npmjs.com/package/ws), by setting the `wsEngine` option for Socket.IO in `laravel-echo-server.json`:
-
-```js
-"socketio": {
-    "wsEngine": "ws"
-}
-```
-
-See <https://github.com/uWebSockets/uWebSockets> for more information.
+µWebSockets has been [officially deprecated](https://www.npmjs.com/package/uws). Currently there is no support for µWebSockets in Socket.IO, but it may have the new [ClusterWS](https://www.npmjs.com/package/@clusterws/cws) support incoming. Meanwhile Laravel Echo Server will use [`ws` engine](https://www.npmjs.com/package/ws) by default until there is another option.
```
Comment (DarkGhostHunter) [inline README.md]: I think it's better now. Simple: there is no `uws` support.

File: README.md (hunk 2 above)
Comment (DarkGhostHunter) [reply README.md]: Fixed.

**Pull request merged**